zanata-war/src/main/webapp/WEB-INF/layout/delete_confirmation_modal.xhtml

```diff
@@ -0,0 +1,38 @@
+<ui:composition xmlns="http://www.w3.org/1999/xhtml"
+  xmlns:ui="http://java.sun.com/jsf/facelets"
+  xmlns:a4j="http://richfaces.org/a4j"
+  xmlns:s="http://jboss.org/schema/seam/taglib">
+
+
+
+  <div class="modal" id="delete-confirmation-modal" tabindex="-1"
+    role="dialog" aria-hidden="true">
+    <div class="modal__dialog l--constrain-medium">
+      <header class="modal__header">
+        <h2 class="modal__title">#{msgs['jsf.DeleteConfirmation']}</h2>
+        <button class="modal__close button--link" data-dismiss="modal"><i
+          class="i i--huge i--cancel"></i></button>
+      </header>
+
+      <div class="modal__content">
+        <div class="message message--highlight l--pad-h-1">
+          #{msgs['jsf.DeleteConfirmation.Message']}
+        </div>
+      </div>
```

Comment:
Just realise this model is used for project and version delete. Perhaps you can pass in a param for the name of the entity to be deleted.  
```
<ui:include src...>
<ui:param name="entityName" value="{project name or version name}" />
</ui:include>
```

Then you can access the param in `delete_confirmation_modal.xhtml` with `#{entityName}`